src/coreclr/jit/lower.h

```diff
@@ -260,13 +260,8 @@ class Lowering final : public Phase
     // return true if this call target is within range of a pc-rel call on the machine
     bool IsCallTargetInRange(void* addr);
 
-<<<<<<< HEAD
 #if defined(TARGET_XARCH) || defined(TARGET_WASM)
```

Comment:
Why do we have `TARGET_WASM` ifdefs here? The whole file is ifdef-ed out.

Reply:
Removed.